src/_packagedcode/pypi.py

```diff
@@ -719,10 +728,6 @@ def parse(cls, location):
         if not dependency_type:
             return
 
-        dependencies = parse_with_dparse2(
```

Comment:
You also need to remove the setup.cfg reference in https://github.com/nexB/python-inspector/pull/31/files#diff-e40758b21617f9a1a66c5bf8a887010a4750163a9ed632d1df23c878b50a7a56R1285 ... line 1285